scripts/gulpfile.factory.js

```diff
@@ -18,7 +18,7 @@ module.exports = function factory({
 
   gulp.task("merge", () => {
     return gulp
-      .src(`./src/**/*.js`)
+      .src([`./src/**/*.js`, `!./src/*.story.js`])
```

Comment:
@mwcz Since you moved story into the demo directory, is this still needed?

Reply:
Good catch!